Makefile

```
@@ -2,7 +2,7 @@ SHELL = /bin/bash
 TAG ?= $(shell git describe --exact-match 2>/dev/null)
 COMMIT = $(shell git rev-parse --short=7 HEAD)$(shell [[ $$(git status --porcelain) = "" ]] || echo -dirty)
 ARO_IMAGE_BASE = ${RP_IMAGE_ACR}.azurecr.io/aro
-INTERMEDIATE_IMAGE_LOCAL = ${INTERMEDIATE_IMAGE_ACR_LOCAL}.azurecr.io/aro
+RP_IMAGE_LOCAL = localhost/aro
```

Can we move this definition to right before the portal/builder images so the context is more apparent? 

Additionally, should we set this variable only if it is already unset? e.g. 
```suggestion
RP_IMAGE_LOCAL ?= localhost/aro
```

---

We can, I'll update it

---

Done 👍🏼